sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -148,6 +149,25 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
     }
   }
 
+  test("verify table size calculation is accurate") {
+    val checkSizeTable = "checkSizeTable"
```

Comment:
super nit: can you move this variable inside `withSQLConf`?

Reply:
Done.